libraries/AP_Math/crc.cpp

```diff
@@ -405,6 +405,20 @@ uint16_t calc_crc_modbus(uint8_t *buf, uint16_t len)
     return crc;
 }
 
+// fletcher 16 implementation
+uint16_t crc_fletcher16(const uint8_t *buffer, uint32_t len) {
+    uint16_t c0 = 0;
+    uint16_t c1 = 0;
+    for (uint32_t i = 0; i < len; i++) {
+        c0 = (c0 + buffer[i]) % 255;
```

Comment:
given the mod 255, can we just declare as uint8_t and not need the % 255 at all?